lib/src/base-styles/_nx-forms.scss

```diff
@@ -39,6 +39,7 @@
 .nx-label, .nx-legend {
   display: block;
   padding: 0;
+  max-width: var(--nx-width-text-wrapping);
```

Comment:
I don't think we've ever considered whether these are supposed to wrap or truncate in the first place. We've always assumed they should/would be short enough for it not to matter.  Which should they do? Or if we support neither and just demand they be short, this property isn't really needed here.

Reply:
> I don't think we've ever considered whether these are supposed to wrap or truncate in the first place. We've always assumed they should/would be short enough for it not to matter. Which should they do? Or if we support neither and just demand they be short, this property isn't really needed here.

IMO they should be short and this is _should_ be unnecessary, but I think that it's safer to add it here. Call it CYA. Should there be an absurdly long label I think that wrapping is more appropriate than truncation.